DOCS-2336 and DOCS-2369 Synthetics CI/CD Testing Edits (#11447)
DataDog/documentation: content/en/synthetics/api_tests/ssl_tests.md

## Overview

> SSL tests allow you to **proactively monitor the validity and expiration of your SSL/TLS certificates** to ensure secure connections between your key services and your users. If your certificate is about to expire or becomes compromised, Datadog sends you an **alert with details on the failure**, allowing you to quickly pinpoint the root cause of the issue and fix it.

Reviewer: Why deleting the bold?
I'm not against it, just want to understand your thoughts

Author: While our Contributing Style Guide (https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) states that bold subjectively points readers to something important, I use bold specifically to call out UI elements. In this Overview section, the bold is fine for action verbs, but I'd rather not dilute the purpose of bold.

SSL tests can run from [managed][1] and [private locations][2] depending on whether you want to monitor certificates of public or internal hosts.

## Permissions

By default, only users with the [Datadog Admin and Datadog Standard roles][3] can create, edit, and delete Synthetic SSL tests. To get create, edit, and delete access to Synthetic SSL tests, upgrade your user to one of these two [default roles][3].

If you have access to the [custom role feature][4], add your user to a custom role that includes `synthetics_read` and `synthetics_write` permissions for Synthetic Monitoring.

## Configuration

> Now that you have chosen to create a [`SSL` test][5], define your test's request.

Reviewer 2: ```suggestion
After choosing to create a [`SSL` test][5], define your test's request.
```
